woocommerce/woocommerce
# Add CRUD data store create and update actions

### All Submissions:






### Changes proposed in this Pull Request:

Adds the create and update actions to crud data store utilities.  This adds those methods to the product attributes data store which is already making use of the CRUD data store.

Closes #33437 .

### How to test the changes in this Pull Request:

1. Try to create a product attribute: `wp.data.dispatch('wc/admin/products/attributes').createProductAttribute({ name: 'Test Attribute' });`
2. Visit `/wp-admin/edit.php?post_type=product&page=product_attributes` and make sure your attribute has been created
3. Find the ID of the attribute (you can pull attributes by `wp.data.select('wc/admin/products/attributes').getProductAttributes({});`)
4. Attempt to update a product attribute via the ID `wp.data.dispatch('wc/admin/products/attributes').updateProductAttribute(3, { name: 'New Attribute Name' });`
5. Check that readme updates make sense

### Other information:




### FOR PR REVIEWER ONLY:
